share/shwap/p2p/bitswap/row_namespace_data.go

```diff
@@ -0,0 +1,112 @@
+package bitswap
+
+import (
+	"fmt"
+
+	blocks "github.com/ipfs/go-block-format"
+	"github.com/ipfs/go-cid"
+
+	"github.com/celestiaorg/rsmt2d"
+
+	"github.com/celestiaorg/celestia-node/share"
+	"github.com/celestiaorg/celestia-node/share/shwap"
+	bitswapb "github.com/celestiaorg/celestia-node/share/shwap/p2p/bitswap/pb"
+)
+
+const (
+	// rowNamespaceDataCodec is a CID codec used for data Bitswap requests over Namespaced Merkle Tree.
+	rowNamespaceDataCodec = 0x7820
```

Comment:
Where does this `const` come from?

Reply:
It is defined in the spec https://github.com/celestiaorg/CIPs/blob/main/cips/cip-19.md#multihashes-and-cids